app/helpers/application_helper.rb

```diff
@@ -7,4 +7,27 @@ def user_image_tag(owner)
     email = owner.nil? ? nil : owner.email
     gravatar_image_tag(email)
   end
+
+  def test_nodes
+    [
+      {id: 1,  name: 'Gru',     ip: '192.168.15.20', network: 'Europe',  master: true,  status: 'ready',    new: false, isSelected: false},
+      {id: 2,  name: 'Kevin',   ip: '192.168.15.30', network: 'Europe',  master: false, status: 'ready',    new: false, isSelected: false},
+      {id: 3,  name: 'Kyle',    ip: '120.158.85.34', network: 'America', master: false, status: 'info',     new: false, isSelected: false},
+      {id: 4,  name: 'Bob',     ip: '120.158.85.20', network: 'America', master: false, status: 'info',     new: false, isSelected: false},
+      {id: 5,  name: 'Stuart',  ip: '220.68.155.90', network: 'Asia',    master: false, status: 'warning',  new: false, isSelected: false},
+      {id: 6,  name: 'Dave',    ip: '192.168.15.55', network: 'Europe',  master: false, status: 'ready',    new: true,  isSelected: false},
+      {id: 7,  name: 'Steve',   ip: '192.168.15.20', network: 'America', master: false, status: 'ready',    new: true,  isSelected: false},
+      {id: 8,  name: 'Donny',   ip: '192.168.15.20', network: 'America', master: false, status: 'ready',    new: true,  isSelected: false},
+      {id: 9,  name: 'Ken',     ip: '220.68.155.93', network: 'Asia',    master: false, status: 'ready',    new: true,  isSelected: false},
+      {id: 10, name: 'Mike',    ip: '220.68.155.94', network: 'Asia',    master: false, status: 'ready',    new: true,  isSelected: false},
+      {id: 11, name: 'Paul',    ip: '220.68.155.96', network: 'Asia',    master: false, status: 'ready',    new: true,  isSelected: false},
+      {id: 12, name: 'Larry',   ip: '220.68.155.97', network: 'Asia',    master: false, status: 'ready',    new: true,  isSelected: false},
+    ]
+  end
```

Comment:
I suggest we jump in after the initial review and fix things before merging. I would prefer our master to be in good shape. Since we don't have a "stable" branch yet, the master will be the first thing someone would try out.